seleniumJavaExperiment/src/test/java/pageObjects/DropDownPage.java

```diff
@@ -0,0 +1,32 @@
+package pageObjects;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+public class DropDownPage {
+    @FindBy(css = ".example h3")
+    WebElement pageTitle;
+
+    @FindBy(id = "dropdown")
+    WebElement dropdown;
+
+    public String getTitleText () {
+        return pageTitle.getText();
+    }
+
+    public void openDropDown(){dropdown.click();}
```

Comment:
When using select, you do not need to open the dropdown. You can directly use select.